client/src/utils/hyperscript/index.ts

```diff
@@ -1,17 +1,11 @@
 import {h} from "@stencil/core";
 import {VNode} from "@stencil/core";
-import {isHTMLTagName} from "is-html-tag-name";
```

Comment:
Are these removals related to this PR?

Reply:
No, but it's good housekeeping

Reply:
I like the house cleaning, we may want to push it to a separate PR in case it leads to different regressions with it.

Reply:
:+1: – I'd move this into a separate PR.